### PR TITLE
Add meetups for Nuemberg/DE, Vienna/AT, Stuttgart/DE

### PR DESCRIPTION
**Pull request by t0r0X**

### CODE REVIEW

File: draft/2022-03-16-this-week-in-rust.md [hunk 1]
```diff
@@ -169,12 +169,18 @@ Rusty Events between 2022-03-09 - 2022-04-06 🦀
     * [**Monthly meetup**](https://www.meetup.com/Seattle-Rust-Meetup/events/gskksrydcfblb/)
 * 2022-03-16 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
     * [**Building a Randomizer**](https://www.meetup.com/Vancouver-Rust/events/283910183/)
+* 2022-03-16 | Wien, AT | [Mob-Programming on Open Source Software](https://www.meetup.com/de-DE/Mob-Programming-on-Open-Source-Software/)
+    * [**The Rustic Mob**](https://www.meetup.com/de-DE/Mob-Programming-on-Open-Source-Software/events/284548235/)
+* 2022-03-17 | Stuttgart, DE | [Rust Community Stuttgart](https://www.meetup.com/de-DE/Rust-Community-Stuttgart/)
```
Comment (andrewpollack) [inline draft/2022-03-16-this-week-in-rust.md:174]: This event added as part of another PR
```suggestion
```

File: draft/2022-03-16-this-week-in-rust.md [hunk 2]
```diff
@@ -169,12 +169,18 @@ Rusty Events between 2022-03-09 - 2022-04-06 🦀
     * [**Monthly meetup**](https://www.meetup.com/Seattle-Rust-Meetup/events/gskksrydcfblb/)
 * 2022-03-16 | Vancouver, BC, CA | [Vancouver Rust](https://www.meetup.com/Vancouver-Rust/)
     * [**Building a Randomizer**](https://www.meetup.com/Vancouver-Rust/events/283910183/)
+* 2022-03-16 | Wien, AT | [Mob-Programming on Open Source Software](https://www.meetup.com/de-DE/Mob-Programming-on-Open-Source-Software/)
```
Comment (andrewpollack) [inline draft/2022-03-16-this-week-in-rust.md:172]: Removing location tag from URL for consistency
```suggestion
* 2022-03-16 | Wien, AT | [Mob-Programming on Open Source Software](https://www.meetup.com/Mob-Programming-on-Open-Source-Software/)
```

File: draft/2022-03-16-this-week-in-rust.md [hunk 3]
```diff
@@ -169,12 +169,18 @@ Rusty Events between 2022-03-09 - 2022-04-06 🦀
...
+    * [**The Rustic Mob**](https://www.meetup.com/de-DE/Mob-Programming-on-Open-Source-Software/events/284548235/)
```
Comment (andrewpollack) [inline draft/2022-03-16-this-week-in-rust.md:173]: Removing location tag from URL for consistency

```suggestion
    * [**The Rustic Mob**](https://www.meetup.com/Mob-Programming-on-Open-Source-Software/events/284548235/)
```

File: draft/2022-03-16-this-week-in-rust.md [hunk 4]
```diff
@@ -169,12 +169,18 @@ Rusty Events between 2022-03-09 - 2022-04-06 🦀
...
+* 2022-03-17 | Stuttgart, DE | [Rust Community Stuttgart](https://www.meetup.com/de-DE/Rust-Community-Stuttgart/)
+    * [**Rust-Meetup**](https://www.meetup.com/de-DE/Rust-Community-Stuttgart/events/lwgrssydcfbwb/)
```
Comment (andrewpollack) [inline draft/2022-03-16-this-week-in-rust.md:175]: This event added as part of another PR

```suggestion
```

File: draft/2022-03-16-this-week-in-rust.md [hunk 5]
```diff
@@ -169,12 +169,18 @@ Rusty Events between 2022-03-09 - 2022-04-06 🦀
...
 * 2022-03-23 | Egg Harbor City, NJ, US | [Neighborhood Math Club](https://www.meetup.com/neighborhood-math-club/)
     * [**The Early Rustacean Gets The Worm!**](https://www.meetup.com/neighborhood-math-club/events/284379020/)
 * 2022-04-05 | Buffalo, NY, US | [Buffalo Rust Meetup](https://www.meetup.com/Buffalo-Rust-Meetup/)
     * [**Buffalo Rust User Group, First Tuesdays**](https://www.meetup.com/Buffalo-Rust-Meetup/events/284342808/)
 * 2022-04-06 | Indianapolis, IN, US | [Indy Rust](https://www.meetup.com/indyrs/)
     * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/284244527/)
+* 2022-04-07 | Nürnberg, DE | [Rust Nuremberg](https://www.meetup.com/de-DE/rust-noris/)
```
Comment (andrewpollack) [inline draft/2022-03-16-this-week-in-rust.md:182]: This event added as part of another PR

```suggestion
```

File: draft/2022-03-16-this-week-in-rust.md [hunk 6]
```diff
@@ -169,12 +169,18 @@ Rusty Events between 2022-03-09 - 2022-04-06 🦀
...
+  * [**Rust Nürnberg online #12**](https://www.meetup.com/de-DE/rust-noris/events/zgfnssydcgbsb/)
```
Comment (andrewpollack) [inline draft/2022-03-16-this-week-in-rust.md:183]: This event added as part of another PR

```suggestion
```

**Review by andrewpollack — CHANGES_REQUESTED** [review]
Thank you for the PR! Two of these events were pulled in as part of another PR, but one of them can be added by this PR. Can you make the suggested removals? Happy to merge once the changes are made!

**Comment by t0r0X** [comment]
> Thank you for the PR! Two of these events were pulled in as part of another PR, but one of them can be added by this PR. Can you make the suggested removals? Happy to merge once the changes are made!

Sure. Thanks for looking through these commits.

**Comment by t0r0X** [comment]
Removals done, merge conflict solved.

**Review by andrewpollack — APPROVED** [review]
LGTM! Thank you for following up with the changes -- merging now 😸

**Pull request merged**